web/src/people.ts

```diff
@@ -28,6 +28,7 @@ export type User = {
     delivery_email: string | null;
     email: string;
     full_name: string;
+    name_with_diacritics_removed?: string; // used for caching result of remove_diacritics.
```

Comment:
Comments documenting a field should go on the line before it.

Reply:
Ah okay, can I then update the comments below on this object (like `// used for fake user objects.`) in a followup commit? I like consistency